static/js/click_handlers.js

```diff
@@ -242,6 +246,16 @@ export function initialize() {
         window.location.href = $(this).attr("href");
     });
 
+    $("body").on("click", "[data-video-original-url]", (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        const video = e.currentTarget;
+        video.src = $(video).attr("data-video-original-url");
+        $(video).removeAttr("data-video-original-url");
+        video.play();
```

Comment:
replace the url here on click and play it. Also, remove the `data-video-original-url` so that this click handler isn't triggered multiple times.